src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+}
```

Comment:
The body styling should not be in the calendar.scss file as it is a global style. It should be moved to a different file such as global.scss.